cmd/traffic_manager/AddConfigFilesHere.cc

```diff
@@ -77,5 +77,6 @@ initializeRegistry()
   configFiles->addFile("splitdns.config", false);
   configFiles->addFile("ssl_multicert.config", false);
   configFiles->addFile("metrics.config", false);
+  configFiles->addFile("ssl_SNI.config", false);
```

Comment:
I feel pretty strongly that we should *not* have upper case letters in the filename however.

Also, this being a new configuration file, I wonder if we should mandate that any new configuration is using the Lua format? @SolidWallOfCode  thoughts?

I'd recommend just **sni.lua**, **sslsni.lua**, **ssl_sni.lua** or perhaps s/.lua/.luaconfig/ if that's where we go. Such a change though (to go to Lua) would require significant rework here, but I think it's worth it to set the stage clear for new configurations.